ArduPlane/quadplane.cpp

```diff
@@ -662,12 +658,13 @@ bool QuadPlane::setup(void)
     case AP_Motors::MOTOR_FRAME_TAILSITTER:
         break;
     default:
-        AP_BoardConfig::config_error("Unknown Q_FRAME_CLASS %u", (unsigned)frame_class.get());
+        AP_BoardConfig::config_error("Unsupported Q_FRAME_CLASS %u", frame_class);
     }
+    ::printf("QuadPlane initialise, class: %u, type: %u\n", frame_class.get(), frame_type.get());
 
     if (tailsitter.motor_mask == 0) {
         // this is a normal quadplane
-        switch (motor_class) {
+        switch ((AP_Motors::motor_frame_class)frame_class) {
```

Comment:
```suggestion
        switch (frame_class) {
```

Prettier without the cast :-)

Reply:
but doesn't compile?